Tester.java

```diff
@@ -8,10 +8,429 @@
  */
```

Comment:
Add alot more comments and print outputs explaining what a specific test does. Thats one of the bullet points for task 5